cdk/src/stacks/backup.ts

```diff
@@ -6,8 +6,10 @@ import { Construct } from 'constructs';
 export class BackupStack extends NestedStack {
   constructor(scope: Construct, id: string, props: NestedStackProps) {
     super(scope, id, props);
-    new CfnInclude(this, 'SuperwerkerTemplate', {
+    const cfnInclude = new CfnInclude(this, 'SuperwerkerTemplate', {
       templateFile: path.join(__dirname, '..', '..', '..', 'templates', 'backup.yaml'),
     });
+
+    cfnInclude.stack.addMetadata('cfn - lint', { config: { ignore_checks: ['E9007', 'EPolicyWildcardPrincipal', 'E1029'] } });
```

Comment:
Just need to remember to delete this one after we have migrated the respective CFN stacks to CDK.

Reply:
Yes, or just not copy over the metadata to the cdk stack ;) 
I think we never did this previously.

Reply:
But at the end of the day we don't really need it, cfn lint is more of a tool when writing plain cloudformation. Cdk does pretty decent stuff and nag on top should be sufficient 👍🏻

Reply:
Yes, but at the moment it is still enabled via a github action and we still have have a few cfn files.